packages/@tailwindcss-standalone/scripts/build.ts

```diff
@@ -0,0 +1,71 @@
+import { $ } from 'bun'
+import { createHash } from 'node:crypto'
+import { mkdir, readFile, writeFile } from 'node:fs/promises'
+import * as path from 'node:path'
+import { fileURLToPath } from 'node:url'
+
+const __dirname = fileURLToPath(new URL('.', import.meta.url))
+
+async function buildForPlatform(triple: string, outfile: string) {
+  // We wrap this in a retry because occasionally the atomic rename fails for some reason
+  for (let i = 0; i < 5; ++i) {
+    try {
+      return await $`bun build --compile --target=${triple} ./src/index.ts --outfile=${outfile}`
+    } catch (err) {
+      if (i < 5) continue
+
+      throw new Error(`Failed to build for platform ${triple}`, { cause: err })
+    }
+  }
+}
+
+async function build(triple: string, file: string) {
+  let start = process.hrtime.bigint()
+
+  let outfile = path.resolve(__dirname, `../dist/${file}`)
+
+  await buildForPlatform(triple, outfile)
+
+  await new Promise((resolve) => setTimeout(resolve, 100))
```

Comment:
Was this delay a workaround of a bug?

Reply:
It was because the `readFile` that follows would occasionally fail unless we waited a little bit. Perhaps it's like a filesystem buffer syncing thing?

I need to give it another test to see if its still a problem.